main.go

```diff
@@ -49,7 +49,7 @@ func init() {
 }
 
 func main() {
-	fdb.MustAPIVersion(620)
+	fdb.MustAPIVersion(610)
```

Comment:
`620` is currently not available in the old fdb lib and I don't wanted to update the fdb lib as part of this PR.